website/docs/api/cli.mdx

```diff
@@ -361,7 +361,7 @@ Module     spacy.language
 File       /path/to/spacy/language.py (line 64)
 ℹ [components.ner.model]
 Registry   @architectures
-Name       spacy.TransitionBasedParser.v3
+Name       spacy.TransitionBasedParser.v1
```

Comment:
Also ?

```suggestion
Name       spacy.TransitionBasedParser.v2
```

Reply:
I wasn't to sure about that, since it's an example with program output.

Reply:
Ah, you're right. I think this doesn't belong in this PR. (On the other hand, it doesn't matter that much either at this point...)